Sources/Sentry/SentryOptions.m

```diff
@@ -23,6 +23,17 @@ - (_Nullable instancetype)initWithDict:(NSDictionary<NSString *, id> *)options
         if (nil != error && nil != *error) {
             return nil;
         }
+
+        // If no user define release, use default.
```

Comment:
I think we have a typo in this comment. I don't get it sorry 😄

Reply:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/1633368/80488814-532dbd80-8935-11ea-81cf-b87e9639284a.png">
yeah sorry I never pushed my own "fix".
Will fix